lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -25,6 +25,7 @@
       </table>
     </div>
     <div class="bottom-bar">
+      <label for="add-field" class="sr">{{add_placeholder}}</label>
       <input type="text" name="add-field" class="add-field" placeholder="{{add_placeholder}}">
```

Comment:
there is a for attribute on the label above but it is currently referencing a non-existent ID on this input.  need to add ```id="add-field``` here.  FYI, on all of these labels, it would be easier to just make the relationship implied by making the form input a child of the ```<label>``` element. Then you wouldn't have to worry about all of this for/ID attribute detail.